jenkins/agent-ctl.sh

```diff
@@ -75,12 +75,21 @@ while true; do
       FAKE=yes
       shift
       ;;
+    --previous)
+      # Currently jenkins-agent-20160926-0059
+      IMAGE_FLAG='--image=jenkins-agent-20160613-2240'
+      ;;
+    --previous-pr)
+      # Currently jenkins-agent-20160926-0000
+      IMAGE_FLAG='--image=jenkins-agent-20160613-1431'
+      PR=yes
+      ;;
     --pr)
       PR=yes
       shift
       ;;
     --base-image)
-      IMAGE_FLAG="--image=${BASE_IMAGE}"
+      IMAGE_FLAG="--image-family=${BASE_IMAGE}"
```

Comment:
is the switch from `image` to `image-family` intentional?

I'm trying to understand this change wrt the documentation for this flag:
> --base-image: create instance with base image instead of family